test/cases/enum.ts

```diff
@@ -6,7 +6,8 @@ export var schema = {
       "enum": ["a", "b", "c"]
     },
     "bar": {
-      "enum": [1, 2, 3]
+      "enum": [1, 2, 3],
+      "tsEnumNames": ["One","Two","Three"]
```

Comment:
>  In TS, the way to do that is to specify a mapping from String => Number (via, for example, tsEnumValues)

The problem then is portability.  If other people are reading your schema declaration without your special ts-only attribute, they need to know that the integers are the values to expect.  That's why I think that fundamentally the values of `enum` need to be the integers, and the additional attribute can attach extra semantic meaning for typescript in particular.  Agree?  Disagree?